test/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ add_chimera_binding(
 
 set_target_properties(test_binding PROPERTIES PREFIX "")
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I haven't dig further, but it's not compiled without it complaining `std::function` is not defined even with `<functional>` included.